charts/prometheus/templates/vpa.yaml

```diff
@@ -1,11 +1,11 @@
-{{- if .Values.server.verticalAutoscaler.enabled -}}
-apiVersion: autoscaling.k8s.io/v1beta2
+{{- if and (.Capabilities.APIVersions.Has "autoscaling.k8s.io/v1") (.Values.server.verticalAutoscaler.enabled) }}
+apiVersion: autoscaling.k8s.io/v1
```

Comment:
Just curious if any document (e.g. in README.md) relates to upgrade (e.g. which versions K8S drop support).

Reply:
I think these are out of our scope,

VPA is disable by default and requires pre-installation of CRD.

https://pkg.go.dev/k8s.io/autoscaler/vertical-pod-autoscaler#section-readme